src/main/kotlin/fi/metatavu/timebank/api/controllers/SynchronizeController.kt

```diff
@@ -97,37 +71,29 @@ class SynchronizeController {
                 }
             }
             logger.info("Finished synchronization with: $synchronized entries synchronized... $duplicates entries NOT synchronized...")
-        } catch (e: Error) {
-            logger.error("Error during synchronization: ${e.localizedMessage}")
-            throw Error(e.localizedMessage)
-        }
-    }
-
-    /**
-     * Synchronizes deleted time entries from Forecast to Time-bank
-     *
-     * @param after YYYY-MM-DD LocalDate
-     * @param forecastPersons list of ForecastPersons
-     * @param forecastTimeEntries list of ForecastTimeEntries
-     */
-    suspend fun synchronizeDeletions(after: LocalDate? = oldestSyncDate, forecastPersons: List<ForecastPerson>, forecastTimeEntries: List<ForecastTimeEntry>) {
 
-        val timeBankTimeEntries = timeEntryController.getEntries(personId = null, before = null, after = after, vacation = false)
+            if (syncDeletedEntries!!) {
```

Comment:
```suggestion
            if (syncDeletedEntries) {
```
Remove non-null assertion as you change syncDeletedEntries from `Boolean?` to `Boolean`